content/posts/2020/07/13/client-upgrades.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2020-07-13T18:32:00
+draft: false
+title: 'Your client and InspIRCd'
+type: 'post'
+---
+
+We are currently working on upgrading our network to [InspIRCd v3](https://www.inspircd.org/2019/05/03/v300-released.html).
+Older versions of some IRC clients (listed below) are non-compliant with the way
```

Comment:
Might be a good idea to point out that the way that InspIRCd v3 emits messages is standards-compliant. Maybe this?

```suggestion
Older versions of some IRC clients (listed below) do not follow the IRC specification correctly and can not handle the way that
```